Add 32 new config and schema files and corresponding extractor functions

aknap opened the pull request:
List of new portals included in this commit:

- 24.hu
- 888.hu
- alfahir.hu
- atlatszo.ro
- demokrata.hu
- eduline.hu
- feol.hu
- forum.portfolio.hu
- foter.ro
- greenfo.hu
- hang.hu
- hvg.hu
- kronikaonline.ro
- kuruc.info
- lelato.transindex.ro (subdomain of transindex.ro)
- magyarnarancs.hu
- maszol.ro
- merce.hu
- neokohn.hu
- nepszava.hu
- nlc.hu
- parameter.sk
- penzcsinalok.transindex.ro (subdomain of transindex.ro)
- pestisracok.hu
- plakatmagany.transindex.ro (subdomain of transindex.ro)
- portfolio.hu
- szekelyhon.ro
- szombat.org
- transindex.ro
- tv.transindex.ro (subdomain of transindex.ro)
- ujszo.com
- vadhajtasok.hu

dlazesz commented:
1. Please leave a space after \# comments
2. Indicate why verify_request is False
3. Unify "main_column" to "main-column"
4. Avoid search string if possible, use \* for joker search string if possible
5. Use daily archive if there is one
6. Use date based search if possible
7. Avoid infinite scrolling in favour of next_page_url if possible
8. Delete unnecessary Hungarian comments (TODOs)
9. Please merge transindex configs where possible

dlazesz commented:
The python file:

1. Compile regex outside of the function (`import re` is missing)
2. Extra newline in 130,1606
3. Missing newline 162,1613
4. Extra space after operator at 506
5. 792 Do not use try-except use if

Should be better to create a separate python file for these functions and a separate test warc file.

aknap commented:
- Fixed listed issues on these sites: 24.hu, atlatszo.ro, demokrata.hu, lelato.transindex.ro, penzcsinalok.transindex.ro, plakatmagany.transindex.ro, transindex.ro, tv.transindex.ro)
- Added config for 4 further 24.hu subdomains (istentudja.24.hu, rangado.24.hu, roboraptor.24.hu, sokszinuvidek.24.hu)
- Added atv.hu configuration
